### ISSUE TITLE
Failed to patch Nx dep-graph for Vue support. 

### ISSUE THREAD

**Issue by mklueh** [ZachJW34/nx-plus#235]
After a while I've tried it again with this package and the new version.

I'm running into a new issue right when I try to create an app:

```
$ node node_modules/@nx-plus/nuxt/patch-nx-dep-graph.js
Failed to patch Nx dep-graph for Vue support. Error: ENOENT: no such file or directory, open '/home/user/workspace/nuxt3-multi-page/node_modules/@nrwl/workspace/src/core/project-graph/build-dependencies/typescript-import-locator.js'
    at Object.openSync (node:fs:585:3)
    at Object.readFileSync (node:fs:453:35)
    at patchNxDepGraph (/home/user/workspace/nuxt3-multi-page/node_modules/@nx-plus/nuxt/patch-nx-dep-graph.js:14:28)
    at Object.<anonymous> (/home/user/workspace/nuxt3-multi-page/node_modules/@nx-plus/nuxt/patch-nx-dep-graph.js:29:1)
    at Module._compile (node:internal/modules/cjs/loader:1103:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1157:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:77:12)
    at node:internal/main/run_main_module:17:47 {
  errno: -2,
  syscall: 'open',
  code: 'ENOENT',
  path: '/home/user/workspace/nuxt3-multi-page/node_modules/@nrwl/workspace/src/core/project-graph/build-dependencies/typescript-import-locator.js'
}

```

**I've tried it with nx version 13.10.3, is that a problem and only 13.0.0 supported?**

**Comment by ichursin** [comment]
looks like, nx currently provide the solution to create graph plugins: https://nx.dev/extending-nx/project-graph-plugins

@nxplus should support it

**Comment by gregveres** [comment]
Just something to keep in mind, I can't tell from your path, but I have run into this a couple of times when using version 13 of nx. I run into it when I forget to go back up to the workspace directory to run yarn install. 
I was down in the app directory and ran yarn install and I recieved the exact same failure message. I went back to the workspace directory and ran yarn install again and everything was fine.

**Comment by ZachJW34** [comment]
It looks like the file we patch was moved in v13.10.0. There is an open PR fixing this
https://github.com/ZachJW34/nx-plus/pull/246

**Comment by ZachJW34** [comment]
It would be nice to move to the official support, but the patch necessary is so simple vs what is required to implement a custom project-graph that I don't imagine I'll be implementing that anytime soon.

**Comment by ZachJW34** [comment]
Closed via #249

**Comment by avxkim** [comment]
Still got the error:

Error: Cannot find module '/Users/USERNAME/Sites/PROJECT_NAME/node_modules/@nx-plus/nuxt/patch-nx-dep-graph.js
